### ISSUE TITLE
dotnet bundle fails to produce output on OS X (presuming *nix platform)

### ISSUE THREAD

**Issue by peterblazejewicz** [madskristensen/BundlerMinifier#130]
### Installed product versions
I've used 2.0.200 version of tool that ships with current aspnet/Templates:
https://github.com/aspnet/Templates/pull/592

### Description
The `dotnet bundle` action fails to produce output on OS X:
```
WebApplicationBasic dotnet bundle
Running with configuration from /Users/piotrblazejewicz/development/WebApplicationBasic/bundleconfig.json
Processing wwwroot/css/site.css
Could not find file '/Users/piotrblazejewicz/development/WebApplicationBasic/\Users\piotrblazejewicz\development\WebApplicationBasic\wwwroot\css\site.css'.
```
The tool stops on this error

### Steps to recreate
1. Create MVC project using tooling version that supports Bundler
2. move content to *nix machine (here OS X)
3. the steps
```
dotnet restore
dotnet build
dotnet bundle
```

### Current behavior
The command quits and produce error message

### Expected behavior
The command finishes and produces content as per configuration file

I assume that is covered by this commit: https://github.com/madskristensen/BundlerMinifier/commit/6613fdcb9886d08eb4e6071af5b6a238cf81b0d5

Thanks!

**Comment by madskristensen** [comment]
Can you please try with version 2.0.208. This issue should have been fixed. We ran a test on Linux and it worked fine

**Comment by peterblazejewicz** [comment]
Sure. Give me a minute

**Comment by madskristensen** [comment]
And btw @peterblazejewicz, thanks for helping out testing. I just created [this wiki](https://github.com/madskristensen/BundlerMinifier/wiki) that you may find interesting

**Comment by peterblazejewicz** [comment]
Confirmed success with 2.0.208:
```
dotnet restore
log  : Restoring packages for /Users/piotrblazejewicz/development/WebApplicationBasic/project.json...
info :   GET https://api.nuget.org/v3-flatcontainer/microsoft.netcore.dotnethostresolver/index.json
info :   OK https://api.nuget.org/v3-flatcontainer/microsoft.netcore.dotnethostresolver/index.json 154ms
info :   GET https://api.nuget.org/v3-flatcontainer/microsoft.netcore.dotnethost/index.json
info :   OK https://api.nuget.org/v3-flatcontainer/microsoft.netcore.dotnethost/index.json 358ms
log  : Restoring packages for tool 'BundlerMinifier.Core' in /Users/piotrblazejewicz/development/WebApplicationBasic/project.json...
info :   GET https://api.nuget.org/v3-flatcontainer/bundlerminifier.core/index.json
info :   OK https://api.nuget.org/v3-flatcontainer/bundlerminifier.core/index.json 593ms
info :   GET https://api.nuget.org/v3-flatcontainer/bundlerminifier.core/2.0.208/bundlerminifier.core.2.0.208.nupkg
info :   OK https://api.nuget.org/v3-flatcontainer/bundlerminifier.core/2.0.208/bundlerminifier.core.2.0.208.nupkg 904ms
log  : Installing BundlerMinifier.Core 2.0.208.
log  : Restoring packages for tool 'Microsoft.AspNetCore.Razor.Tools' in /Users/piotrblazejewicz/development/WebApplicationBasic/project.json...
log  : Restoring packages for tool 'Microsoft.AspNetCore.Server.IISIntegration.Tools' in /Users/piotrblazejewicz/development/WebApplicationBasic/project.json...
info : Committing restore...
log  : Writing lock file to disk. Path: /Users/piotrblazejewicz/development/WebApplicationBasic/project.lock.json
log  : /Users/piotrblazejewicz/development/WebApplicationBasic/project.json
log  : Restore completed in 6110ms.

NuGet Config files used:
    /Users/piotrblazejewicz/.nuget/NuGet/NuGet.Config

Feeds used:
    https://api.nuget.org/v3/index.jso
```
later:
```
dotnet bundle
Running with configuration from /Users/piotrblazejewicz/development/WebApplicationBasic/bundleconfig.json
Processing wwwroot/css/site.min.css
  Bundled
  Minified
Processing wwwroot/js/site.min.js
  Bundled
  Minified
```
Thanks!

**Comment by peterblazejewicz** [comment]
Fixed with `2.0.208` version (update your deps in project.json)
